sumologic/sumologic.py

```diff
@@ -109,11 +109,6 @@ def put(self, method, params, headers=None, version=None):
         return r
 
     # Logs Search
-    def search(self, query, fromTime=None, toTime=None, timeZone='UTC'):
```

Comment:
in normall circumstances, ideally first that method should throw deprecation warning so that customers using it can be notified to make appropriate changes in the code. 
Do we want to close the support instantaneously? 
If yes, shouldn't the old method throw proper exception so that customer knows what to do?

Reply:
the endpoint was to be shutdown on March 1 at which point any request will generate an error. Would something like this work or should this just be an exception? 
`    def search(self, query, fromTime=None, toTime=None, timeZone='UTC'):
        # params = {'q': query, 'from': fromTime, 'to': toTime, 'tz': timeZone}
        # r = self.get('/logs/search', params)
        warnings.warn("""On 1 Mar 2023, support ends for the old search API: api/v1/logs/search. 
         Any API calls made to the older endpoint will fail after the endpoint is removed. 
         Users should migrate to the newer search jobs API released in 2017. 
         https://help.sumologic.com/docs/api/search-job/.""")
        return json.loads(r.text)`

Reply:
yes looks good